op-node/flags/flags.go

```diff
@@ -44,23 +44,24 @@ var (
 		Usage:   fmt.Sprintf("Predefined network selection. Available networks: %s", strings.Join(chaincfg.AvailableNetworks(), ", ")),
 		EnvVars: prefixEnvVars("NETWORK"),
 	}
+	/* Optional Flags */
 	RPCListenAddr = &cli.StringFlag{
 		Name:    "rpc.addr",
 		Usage:   "RPC listening address",
 		EnvVars: prefixEnvVars("RPC_ADDR"),
+		Value:   "127.0.0.1",
 	}
 	RPCListenPort = &cli.IntFlag{
 		Name:    "rpc.port",
 		Usage:   "RPC listening port",
 		EnvVars: prefixEnvVars("RPC_PORT"),
+		Value:   8545,
```

Comment:
This is the same default value as op-geth will use so will make port conflicts very common.  I'd suggest we pick a different default (e.g. 9545).

Reply:
Switched. Annoyingly we default to 8545 in op-service. Maybe we need to change that.